Source/JavaScriptCore/inspector/JSInjectedScriptHost.cpp

```diff
@@ -227,6 +227,7 @@ JSValue JSInjectedScriptHost::subtype(JSGlobalObject* globalObject, CallFrame* c
             || object->inherits<JSUint8ClampedArray>()
             || object->inherits<JSUint16Array>()
             || object->inherits<JSUint32Array>()
+            || object->inherits<JSFloat16Array>()
```

Comment:
FYI there's a utility `Array.isTypedArray` defined in `Source/WebInspectorUI/UserInterface/Base/Utilities.js` that could use updating like this as well :)

Reply:
Done.